docs/source/refs/issues.rst

```diff
@@ -36,35 +36,6 @@ over stepping different parts of the simulation app. However, at this point, the
 timeline for this feature request.
 
 
-Non-determinism in physics simulation
--------------------------------------
-
-Due to GPU work scheduling, there's a possibility that runtime changes to simulation parameters
-may alter the order in which operations take place. This occurs because environment updates can
-happen while the GPU is occupied with other tasks. Due to the inherent nature of floating-point
-numeric storage, any modification to the execution ordering can result in minor changes in the
-least significant bits of output data. These changes may lead to divergent execution over the
-course of simulating thousands of environments and simulation frames.
-
-An illustrative example of this issue is observed with the runtime domain randomization of object's
-physics materials. This process can introduce both determinancy and simulation issues when executed
-on the GPU due to the way these parameters are passed from the CPU to the GPU in the lower-level APIs.
-Consequently, it is strongly advised to perform this operation only at setup time, before the
-environment stepping commences.
-
-For more information, please refer to the `PhysX Determinism documentation`_.
-
-In addition, due to floating point precision, states across different environments in the simulation
-may be non-deterministic when the same set of actions are applied to the same initial
-states. This occurs as environments are placed further apart from the world origin at (0, 0, 0).
-As actors get placed at different origins in the world, floating point errors may build up
-and result in slight variance in results even when starting from the same initial states. One
-possible workaround for this issue is to place all actors/environments at the world origin
-at (0, 0, 0) and filter out collisions between the environments. Note that this may induce
-a performance degradation of around 15-50%, depending on the complexity of actors and
-environment.
```

Comment:
So did they fix this stuff and now we have deterministic behavior? Or it was just the seeding all along?

Reply:
The PhysX team said they fixed the determinism a few releases ago. The main thing was our seeding issue, which wasn't done correctly.